client/components/card/style.scss

```diff
@@ -50,14 +50,22 @@
 	}
 }
 
+
 // Clickable Card
 .card__link-indicator {
 	color: lighten( $gray, 20% );
 	display: block;
 	height: 100%;
 	position: absolute;
-		top: 0;
-		right: 16px;
+	top: 0;
+	right: 16px;
+
+	html[dir="rtl"] & {
```

Comment:
Why not simply `html[dir="rtl"] &.gridicons-chevron-right`?

Reply:
I'd need to use `html[dir="rtl"] & &.gridicons-chevron-right`, but I think this is more readable.
The first standalone `&` translates to `html[dir="rtl"] .card__link-indicator`